app/src/main/java/asd/org/ahorcado/dal/DataBaseHelper.java

```diff
@@ -0,0 +1,115 @@
+package asd.org.ahorcado.dal;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteException;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.sql.SQLException;
+
+import asd.org.ahorcado.BuildConfig;
+
+/**
+ * Created by Huber on 17/04/2016.
+ */
+public class DataBaseHelper extends SQLiteOpenHelper {
+    private Context mycontext;
+    private static String DB_NAME = "Ahorcado.db";
+    private static String DB_PATH ="/data/data/"+ BuildConfig.APPLICATION_ID+"/databases/";
+    public SQLiteDatabase myDataBase;
+
+    public DataBaseHelper(Context context) throws IOException {
+        super(context,DB_NAME,null,1);
+        this.mycontext=context;
+        boolean dbexist = checkdatabase();
+        if (dbexist) {
+            System.out.println("Database exists");
```

Comment:
These Systems.out.println were made just for debugging?
Or it displays them in somewhere